source/localizable/tutorial/simple-component.md

```diff
@@ -158,6 +158,10 @@ Let's call this action `toggleImageSize`
 
 Clicking the anchor element will send the action to the component.
 Ember will then go into the `actions` hash and call the `toggleImageSize` function.
+
+An "actions hash" is an object in the component that contains functions.
+These functions are called when various UI actions occur, stemming mostly from user interaction such as a click event.
```

Comment:
The phrasing might be a bit tricky for ESL readers, can we tweak it to something more along the lines of "the functions are called when the user interacts with the UI, such as clicking"?

I've separately opened an issue for adding UI to the glossary.